docs/installation/compatibility.md

```diff
@@ -0,0 +1,53 @@
+# Compatibility Matrix
+
+This page provides information about Liqo's compatibility with different Kubernetes providers.
+
+```{admonition} Note
+While the following list includes providers that we have specifically tested, Liqo should work with any Kubernetes-compliant distribution, although some provider-specific configurations might be required.
+```
+
+## Legend
+
+- ✅ Fully supported - All features work as expected
```

Comment:
I think this icon may give problems to readability in table. What about using the same icon with different colors? Fully support green, mostly support blue and partial support yellow?